smarts/core/tests/test_smarts_reset.py

```diff
@@ -63,7 +63,7 @@ def neighborhood_vehicle_poses(agent_obs):
     scenario = next(scenarios)
 
     seed(42)
-    obs = smarts.reset(scenario)
+    obs, dones = smarts.reset(scenario)
```

Comment:
keep the reset interface the same and leave changing reset interface to another issue (another milestone)